x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/trigger/schedule/tool/CronEvalTool.java

```diff
@@ -31,7 +30,7 @@ public static void main(String[] args) throws Exception {
     private static final DateTimeFormatter UTC_FORMATTER = DateTimeFormat.forPattern("EEE, d MMM yyyy HH:mm:ss")
         .withZone(DateTimeZone.UTC).withLocale(Locale.ROOT);
     private static final DateTimeFormatter LOCAL_FORMATTER = DateTimeFormat.forPattern("EEE, d MMM yyyy HH:mm:ss Z")
-        .withZone(DateTimeZone.forTimeZone(TimeZone.getDefault()));
+        .withZone(DateTimeZone.forTimeZone(null));
```

Comment:
minor: I think it would be a bit more obvious to explicitly call `DateTimeZone.getDefault()` instead of `null`.
Since that is what Joda does with the `null` value.
http://joda-time.sourceforge.net/apidocs/src-html/org/joda/time/DateTimeZone.html#line.301

Reply:
Doh! the recommendation is forbidden API ..never mind.